interface/billing/new_payment.php

```diff
@@ -123,302 +119,411 @@
 //==============================================================================
 //===============================================================================
 ?>
+<!DOCTYPE html>
 <html>
 <head>
-<?php if (function_exists('html_header_show')) {
-    html_header_show();
-} ?>
-
-
+    <?php Header::setupHeader(['bootstrap', 'datetime-picker']);?><!--<script type="text/javascript" src="../../library/dialog.js?v=<?php //echo $v_js_includes; ?>"></script>
```

Comment:
Remove bootstrap and bring in others (i think testformat.js)

Reply:
done